### DISCUSSION TITLE
Disable f,F,t,T motion

### DISCUSSION THREAD

**Discussion by armaninux** [folke/flash.nvim#270] [Q&A]
I'd like to use flash.nvim as a replacement for / searches only and I'd like to retain Neovim's regular f,F,t,T behavior. How do I achieve this?
I tried the following to no avail:
`lua require('flash').setup({char = {enabled = false}})`

(Bear with me, I'm a writer, not a coder)

**Comment by myypo** [comment]
Hey. This should work.

`require("flash").setup({
            modes = {
                char = {
                    enabled = false
                }
            }
        })`

**Reply by armaninux** [reply]
much appreciated!
